# Errors reported by the parser (`wikitext/error`)

ğŸ’¼ This rule is enabled in the following configs: ğŸŒ `inherited`, âœ… `recommended`.

<!-- end auto-generated rule header -->

These are problems reported by the upstream parser [wikiparser-node](https://github.com/bhsd-harry/wikiparser-node).

## Rule Details

Examples of **incorrect** code for this rule:

```wikitext
ä¸­æ–‡http://example.com

{{T|arg=something}

{ç®€ä½“}-

[[a]

[[a|b]]]

[http://example.com [tag]text]

[ http://example.com example]

<a>

{{{a}}}

<gallery width=200></gallery>

<br style=filter:none>

<i></i id=x>

{| id=x | class=y
|}

{| style="" style=""
|}
```

Examples of **correct** code for this rule:

```wikitext
[[a|[b]]]

x<y
```
