# `http` or `https` unused as a link protocol (`wikitext/lonely-http`)

ğŸ’¼ This rule is enabled in the following configs: ğŸŒ `inherited`, âœ… `recommended`.

<!-- end auto-generated rule header -->

## Rule Details

Examples of **incorrect** code for this rule:

```wikitext
ä¸­æ–‡http://example.com
```

Examples of **correct** code for this rule:

```wikitext
http://example.com

[https://example.com]

[https://example.com example]

http:
```
