# `http` or `https` unused as a link protocol (`wikitext/lonely-http`)

💼 This rule is enabled in the following configs: 🌐 `inherited`, ✅ `recommended`.

<!-- end auto-generated rule header -->

## Rule Details

Examples of **incorrect** code for this rule:

```wikitext
中文http://example.com
```

Examples of **correct** code for this rule:

```wikitext
http://example.com

[https://example.com]

[https://example.com example]

http:
```
